### DISCUSSION TITLE
Welcome to ShellHub Discussions!

### DISCUSSION THREAD

**Discussion by gustavosbarreto** [shellhub-io/shellhub#572] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by otavio** [comment]
Hello all, I am Otavio Salvador, and I am an embedded systems lover. Professionally, I am the CTO of O.S. Systems, and I am glad to be part of the ShellHub community.

**Comment by sixhills** [comment]
Greetings.  I'm Mike from the UK with a programming and system admin background. I'm a user of ShellHub and find it great for administering Raspberry Pi-type machines behind NAT and firewalls.  I have very little experience of Go but I'm having fun playing with the code to add a few minor enhancements.
